sdk/storage/azure-storage-common/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0-beta.1
```

Comment:
See comment about using a version.h file above.  There needs to be a version.h file in the `inc/azure/<service>/` folder for each service